en/simulation/ignition_gazebo.md

```diff
@@ -56,12 +56,114 @@ In order to run the simulation without running the ignition gazebo gui, one can
 HEADLESS=1 make px4_sitl gz_x500
 ```
 
-In order to increase the verbose output, `VERBOSE_SIM=1` can be used:
+## Usage/Configuration Options
```

Comment:
FYI I was in two minds about this structure. I like what [Gazebo doc does](http://docs.px4.io/main/en/simulation/gazebo.html#usage-configuration-options) in that it lists all the things you can do as headings. I like what you have done in that it is completely obvious how you would extend this document.

Anyway, I prefer this topic name - it is now JUST about setting environment variables. I simplified the text and moved the examples under the same heading.

I moved adding new worlds/models out because that is no longer configuration.